test/service.spec.js

```diff
@@ -1,16 +1,48 @@
 // Generated by CoffeeScript 1.12.3
 (function() {
-  var AWS, MockService, helpers, metadata,
+  var AWS, MockService, helpers, metadata, MockServiceWithErrorCodeMapping
     hasProp = {}.hasOwnProperty;
 
   helpers = require('./helpers');
 
   AWS = helpers.AWS;
 
   MockService = helpers.MockService;
+  MockServiceWithErrorCodeMapping = helpers.MockServiceWithErrorCodeMapping;
 
   metadata = require('../apis/metadata.json');
 
+  describe('AWSQuery compatible error codes', function () {
+    var service;
+    var mapping = { ServiceError: { code: 'Custom.ErrorCode' } };
+    beforeEach(function() {
+      var CustomService = MockServiceWithErrorCodeMapping(mapping);
```

Comment:
Let's use common JavaScript naming conventions for variables, e.g. `customService` (camel-case with lower-case first letter)

Also prefer `let` over `var` since it has better scope

Reply:
`CustomService` is a class. Following the convention that is already used in the project.

Reply:
Using var instead of let